manifests/snmp_listen_ip.pp

```diff
@@ -0,0 +1,13 @@
+define squid::snmp_listen_ip (
```

Comment:
Is it valid squid configuration to specify `snmp_incoming_address` more than once?  If not, maybe this would be better as a class instead of defined type.